popup/OpenClosed.md

```diff
@@ -0,0 +1,101 @@
+                            Current_t==7 && calcHour =="13" || CalcHour == "24", "Open",iif(
+                                Current_t==8 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                    Current_t==9 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                        Current_t==10 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                            Current_t==11 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                Current_t==12 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                    Current_t==13 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                        Current_t==14 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                            Current_t==15 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                                Current_t==16 && calcHour =="13" || CalcHour == "24" || CalcHour=="8", "Open",iif(
+                                                                    Current_t==17 && calcHour =="13" || CalcHour == "24", "Open",iif(
+                                                                        Current_t==18 && calcHour =="13" || CalcHour == "24", "Open",iif(
+                                                                            Current_t==19 && calcHour != "24", "Closed",iif(
+                                                                                Current_t==20 && calcHour != "24", "Closed",iif(
+                                                                                    Current_t==21 && calcHour != "24", "Closed",iif(
+                                                                                        Current_t==22 && calcHour != "24", "Closed",iif(
+                                                                                            Current_t==0 && calcHour != "23", "Closed", "Call")))))))))))))))))))))));
+
+#if day or hours is other, call the park/store to find out when they are open
+var Status = iif(
+    Day_open== "Other" && CalcHour =="Open", "Call " + $feature.PHONE +" for hours of operation",iif(
+		CalcHour == "Other" && Day_open == "Open", "Open Today. Call " + $feature.phone + " for hours", iif(
+			Day_open == "Open" && Hour_open == "Open", "Currently Open",iif(
+			    CalcDay == "Other" && CalcHour == "Other", "Call " + $feature.PHONE +" for hours of operation",iif(
+			        Hour_open == "Closed", "Curently Closed", iif(
+			            Day_open == "Closed", "Currently Closed", "Call " + $feature.phone + " for hours" ))))));
+
+Return Status}
+
+#apply function to your features
+Open($feature.OPERDAYS,$feature.OPERHOURS)
```

Comment:
It would be good to reference the field names first in the expression so the user doesn't have to scroll down or try to find where they should be using their field names. If you do something like the following, it would be easier for them IMO.

```suggestion
// At the top of the expression
var daysField = $feature.OPERDAYS;
var hoursField = $feature.OPERHOURS;

// function declaration
// .. 

// apply function to your features
OpenOrClosed(daysField, hoursField)
```